packages/flutter/test/cupertino/dialog_test.dart

```diff
@@ -2063,6 +2063,7 @@ Widget createAppWithCenteredButton(Widget child) {
 }
 
 
+@pragma('vm:entry-point')
```

Comment:
Is that because we have a member we look up by name in C so we also need to be able to lookup the class by name (and it's not automatically made an entrypoint)?

Reply:
IIRC, at some point the class gets looked up by name as a type and then used as a target for the method lookup, so yeah, it's a class access in native code, and no there's no automatic entrypoint created (though if we want to automatically annotate classes if static methods or generative constructors are annotated, we could consider that as future work).